receiver/LoraToGPSServer/device/device.go

```diff
@@ -322,31 +332,174 @@ func irnasParseSingle(data dataInterface) (*Data, error) {
 	return dataParsed, nil
 }
 
+
+
+
+func G62Parse(data *DataUpPayload) ([]*Data, error) {
+
+	dataParsed := &Data{
+		Valid: true,
+		Attr:  map[string]string{},
+	}
+
+	// Non GPS data.
+	if data.FPort != 1 {
+		dataParsed.Valid = false
+		if os.Getenv("DEBUG") == "1" {
+			log.Printf("skipping non gps data, fport:%+v", data.FPort)
+		}
+		return []*Data{dataParsed}, nil
+	}
+
+	lat, ok := data.Object["latitudeDeg"]
+	if !ok {
+		return nil, errors.New("data doesn't contain lat")
+	}
+	lon, ok := data.Object["longitudeDeg"]
+	
+	if !ok {
+		return nil, errors.New("data object doesn't contain lon")
+	}
+	//data.DevEUI=data.DeviceInfo.DevEui
+
+	dataParsed.Time = int64( data.Time.Unix())
```

Comment:
are you sure that the gps data doesn't also include the time for these new tags? Usually the gps data also icnludes the time.

Reply:
yes, its in data.time

Reply:
i use that. its data.Time dont date.Time

Reply:
oooh, sorry , correct